studybuddies/src/app/services/token-interceptor.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { TokenInterceptorService } from './token-interceptor.service';
+
+describe('TokenInterceptorService', () => {
+  let service: TokenInterceptorService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(TokenInterceptorService);
+  });
+
+  it('should be created', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7163724)
```suggestion
  it("should be created", () => {
```